src/legacy/dom-module.html

```diff
@@ -57,6 +75,7 @@
         modules[id] = this;
         lcModules[id.toLowerCase()] = this;
         this._styleOutsideTemplateCheck();
+        this._setAssetpath();
```

Comment:
I'm not sure it's worth it to always set this attribute and it's likely a small performance cost. Maybe instead we can give `dom-module` an imperative api that returns the assetpath?